src/main/scala/org/constellation/consensus/RoundManager.scala

```diff
@@ -177,6 +169,28 @@ class RoundManager(config: Config)(implicit dao: DAO) extends Actor with ActorLo
 
   }
 
+  private[consensus] def returnTransactionsToPending(transactionsToReturn: Seq[String],
+                                                     roundId: RoundId): Unit = {
+    transactionsToReturn.toList
+      .map(dao.transactionService.lookup(_, TransactionStatus.InConsensus))
```

Comment:
```
transactionsToReturn.toList
  .traverse(dao.transactionService.lookup(_, TransactionStatus.InConsensus))
  .map(_.flatten)
  .map(_.map(dao.transactionService.put(_, TransactionStatus.Pending)))
  .flatMap(_.sequence)
  ...
```

But in my opinion it would be better to hide the implementation inside the tx service and trigger only:
```
dao.transactionService.backFromConsensus(transactionsToReturn.toList)
```